engine/src/main/java/io/camunda/zeebe/streamprocessor/ProcessingStateMachine.java

```diff
@@ -157,8 +148,7 @@ public ProcessingStateMachine(
     recordValues = context.getRecordValues();
     logStreamReader = context.getLogStreamReader();
     logStreamWriter = context.getLogStreamWriter();
-    logStream = context.getLogStream();
-    zeebeState = context.getZeebeState();
+    final LogStream logStream = context.getLogStream();
```

Comment:
:wrench: this is only used to get the partitionId. You can merge this with that (line 161)